### DISCUSSION TITLE
How long did this take?

### DISCUSSION THREAD

**Discussion by BryceG2088** [arlbibek/dVoting#7] [Q&A] [answered]
Hello, I am hoping to make a blockchain-based voting app for a group semester project in my High school computer science class. The group will be me and two other people, all of whom are highly proficient in Java, Python, and web development. We would all be brand new to blockchain development. Do you think a project of similar scope to yours here could be accomplished with this time constraint?

**Comment by arlbibek** [comment, marked as answer]
It took me about 2 months, the first month I researched and coded in the second one. I started the project with zero knowledge of JavaScript and/or Blockchain and I still haven't figured it out. 
It seems like it will take you guys **about a month** to build something similar to this project or better one depending upon your dedication and work that you put in.
